quactography/visu/dist_prob_edge.py

```diff
@@ -53,7 +73,7 @@ def plot_distribution_of_probabilities_edge(
     # print(max(dist_binary_prob, key=dist_binary_prob.get))
     # bin_str = list(map(int, max(dist_binary_prob, key=dist_binary_prob.get)))
     # bin_str_reversed = bin_str[::-1]
-    # bin_str_reversed = np.array(bin_str_reversed)
+    # bin_str_reversed = numpy.array(bin_str_reversed)

```

Comment:
Change back and modify import for `import numpy as np`

Reply:
Also if this is all commented code, remove